api/chat.go

```diff
@@ -53,7 +53,13 @@ func configGinChatRouter(router *gin.RouterGroup) {
 		if chat.Msg == "" || len(chat.Msg) > maxMessageLength {
 			return
 		}
-		if dao.IsUserCooledDown(fmt.Sprintf("%v", tumLiveContext.User.ID)) {
+		isCooledDown, err := dao.IsUserCooledDown(fmt.Sprintf("%v", tumLiveContext.User.ID))
+		if err != nil {
+			sentry.CaptureException(errors.New("could not determine whether user has cooled down"))
+			ctx.(*gin.Context).AbortWithStatus(http.StatusInternalServerError)
```

Comment:
This could kill the WebSocket. I think, in case of an error we should either just return or wait for #173 to be merged and send back a "something went wrong" message. 
But since this should be a very rare edge case a return should be fine

Reply:
I just merged with #173 from `dev`, we should be good to go now.